modules/dxp/apps/segments/segments-experiment-web/src/main/java/com/liferay/segments/experiment/web/internal/util/SegmentsExperimentCookieCleaner.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.segments.experiment.web.internal.util;
+
+import com.liferay.portal.kernel.cookies.CookiesManagerUtil;
```

Comment:
@regiCesar21 In the other task we want to reduce usages of portal kernel utils to remove blocking ServiceProxyFactory usages, so please avoid use kernel utils that point to OSGi service in modules as much as possible. Since all these 3 classes are in the same module, you should be able to make it a simple component.